.gitlab-ci.yml

```diff
@@ -114,7 +115,7 @@ test_live_images:
     URL: ${LIVE_DOMAIN}
   script:
     - pull_artifact_from_s3
-    - test_site_links "images" "${URL}" "True" "False" "True"
+    - test_site_links "images" "${URL}" "True" "True" "True"
```

Comment:
These "True" args are: for check-all, check-external, verbose output. good to have on for the first little while